source/_dashboards/entities.markdown

```diff
@@ -110,7 +110,7 @@ double_tap_action:
   type: map
 {% endconfiguration %}
 
-## Special Row Elements
+## Special row elements
 
 Rather than only displaying an entity's state as a text output, the Entities card supports multiple special rows for buttons, attributes, web links, dividers and sections, etc.
```

Comment:
```suggestion
Rather than only displaying an entity's state as a text output, the entities card supports multiple special rows for buttons, attributes, web links, dividers and sections, etc.
```